src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/OverlayList.js

```diff
@@ -0,0 +1,150 @@
+// @flow
+import Portal from 'react-portal';
+import {observer} from 'mobx-react';
+import React from 'react';
+import {action, computed, observable} from 'mobx';
+import Backdrop from '../Backdrop';
+import Option from './Option';
+import overlayListStyles from './overlayList.scss';
+import type {OverlayListDimensions} from './types';
+import OverlayListPositioner from './OverlayListPositioner';
+
+@observer
+export default class OverlayList extends React.PureComponent {
+    props: {
+        isOpen: boolean,
+        children?: React.Element<*>,
+        onRequestClose?: () => void,
+        /** The top coordinate relative to which the list will be positioned **/
+        anchorTop: number,
+        /** The left coordinate relative to which the list will be positioned **/
+        anchorLeft: number,
+        /** The width of the element relative to which the list will be positioned **/
+        anchorWidth: number,
+        /** The width of the element relative to which the list will be positioned **/
+        anchorHeight: number,
+        /** The index of the child element which will be centered relative to the anchor **/
+        centeredChildIndex: number,
+    };
+
+    static defaultProps = {
+        isOpen: false,
+        centeredChildIndex: 0,
+        anchorTop: 0,
+        anchorLeft: 0,
+        anchorWidth: 0,
+        anchorHeight: 0,
+    };
+
+    @observable offsetHeight: number;
+    @observable offsetWidth: number;
+    @observable centeredChildRelativeTop: number;
+    list: HTMLElement;
+    scrollTop: number;
+
+    componentDidMount() {
+        window.addEventListener('blur', this.requestClose);
+        window.addEventListener('resize', this.requestClose);
```

Comment:
Yes this is the behavior of the native select and also the behavior requested from the designers.